gen/dcos-config.yaml

```diff
@@ -985,23 +985,30 @@ package:
 
       ssl_certificate includes/snakeoil.crt;
       ssl_certificate_key includes/snakeoil.key;
-  - path: /etc/adminrouter-tls.conf
+  - path: /etc/adminrouter-tls-master.conf
     content: |
       # Ref: https://github.com/cloudflare/sslconfig/blob/master/conf
       # Modulo ChaCha20 cipher.
+{% switch adminrouter_tls_cipher_override %}
+{% case "false" %}
       ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
+{% case "true" %}
+      ssl_ciphers {{ adminrouter_tls_cipher_suite }};
+{% endswitch %}
       ssl_prefer_server_ciphers on;
       # To manually test which TLS versions are enabled on a node, use
       # `openssl` commands.
       #
       # See comments on https://jira.mesosphere.com/browse/DCOS-13437 for more
       # details.
-{% switch adminrouter_tls_1_0_enabled %}
-{% case "true" %}
-      ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-{% case "false" %}
-      ssl_protocols TLSv1.1 TLSv1.2;
-{% endswitch %}
+      ssl_protocols {{ adminrouter_tls_version_override }};
+  - path: /etc/adminrouter-tls-agent.conf
+    content: |
+      # Note that Agent Admin Router only servers cluster-internal clients. Hence,
```

Comment:
`s/servers/serve`

If that is too much pain to change then let's punt on it

Reply:
Fixed.